src/geoarrow/geoarrow.h

```diff
@@ -475,8 +475,7 @@ GeoArrowErrorCode GeoArrowArrayWriterInitFromSchema(struct GeoArrowArrayWriter*
 ///
 /// Returns EINVAL for precision values that are not valid or if the writer
 /// is not writing to WKT. Must be called before GeoArrowArrayWriterInitVisitor().
-/// A precision value of -1 indicates that the largest precision that will roundtrip
-/// an ordinate value will be used. The default precision value is -1.
+/// The default precision value is 16. See GeoArrowWKTWriter for details.
```

Comment:
This -1 does not actually work? 
(I remember that I was looking into it last week, because GEOS has something similar and ryu should have the option to trim automatically trailing 0's)

Reply:
Or the conclusion was that trimming is actually is by default, right? (GEOS has an option for it (and which recently was switched to be on by default), but here we just always trim trailing zeros. The precision then only determines how many decimals to show if they would _not_ be trimmed)